forum/users/views.py

```diff
@@ -10,7 +10,21 @@
     TokenObtainPairView as BaseTokenObtainPairView,
     TokenRefreshView as BaseTokenRefreshView
 )
-from rest_framework_simplejwt.tokens import RefreshToken
+from rest_framework import status
```

Comment:
Looks like a duplication of functionality, please check the imports section.

Reply:
Fixed.